WeakAuras/WeakAuras.lua

```diff
@@ -1755,15 +1208,15 @@ function WeakAuras.UnloadAll()
   end
   wipe(timers);
 
-  for id in pairs(conditionChecksTimers.recheckTime) do
```

Comment:
Instead of making conditionChecksTimers and checkdConditions globally available, I'd prefer if there would be functions:

Rename/UnloadAll/Unload/Delete prefixed with Conditions (either in its own table or just in the method name).